SteamKit2/SteamKit2/Steam/CDNClient.cs

```diff
@@ -348,14 +351,25 @@ public async Task<DepotManifest> DownloadManifestAsync( uint depotId, ulong mani
         /// <exception cref="System.ArgumentNullException"><see ref="server"/> was null.</exception>
         /// <exception cref="HttpRequestException">An network error occurred when performing the request.</exception>
         /// <exception cref="SteamKitWebRequestException">A network error occurred when performing the request.</exception>
-        public async Task<DepotManifest> DownloadManifestAsync( uint depotId, ulong manifestId, Server server, string? cdnAuthToken, byte[]? depotKey, Server? proxyServer = null )
+        public async Task<DepotManifest> DownloadManifestAsync( uint depotId, ulong manifestId, ulong manifestRequestCode, Server server, string? cdnAuthToken, byte[]? depotKey, Server? proxyServer = null )
         {
             if ( server == null )
             {
                 throw new ArgumentNullException( nameof( server ) );
             }
 
-            var manifestData = await DoRawCommandAsync( server, string.Format( "depot/{0}/manifest/{1}/5", depotId, manifestId ), cdnAuthToken, proxyServer ).ConfigureAwait( false );
+            string url;
+
+            if ( manifestRequestCode > 0 )
+            {
+                url = $"depot/{depotId}/manifest/{manifestId}/5/{manifestRequestCode}";
```

Comment:
What does the 5 mean?

Reply:
Manifest version as far as I know? I could move it to a const.